### PR TITLE
Add student info

### PR DESCRIPTION
**Pull request by jiachen247**
This feature was requested as avengers could not see the students name when grading.

### CODE REVIEW

File: lib/cadet/assessments/assessments.ex [hunk 1]
```diff
@@ -488,16 +488,17 @@ defmodule Cadet.Assessments do
       |> where(submission_id: ^id)
       |> join(:inner, [a], q in assoc(a, :question))
       |> join(:left, [a, _], g in assoc(a, :grader))
-      |> preload([_, q, g], question: q, grader: g)
+      |> join(:inner, [a, _, _], s in Submission, a.submission_id == s.id)
```
Comment (indocomsoft) [inline lib/cadet/assessments/assessments.ex]: Prefer `|> join (:inner, [a, _, _], s in assoc(a, :submission))`

File: lib/cadet/assessments/assessments.ex [hunk 2]
```diff
@@ -488,16 +488,17 @@ defmodule Cadet.Assessments do
...
+      |> join(:inner, [a, _, _, s], st in User, s.student_id == st.id)
```
Comment (indocomsoft) [inline lib/cadet/assessments/assessments.ex]: Prefer `|> join(:inner, [a, ..., s], st in assoc(s, :student))`

File: lib/cadet/assessments/assessments.ex [hunk 3]
```diff
@@ -488,16 +488,17 @@ defmodule Cadet.Assessments do
       |> where(submission_id: ^id)
       |> join(:inner, [a], q in assoc(a, :question))
       |> join(:left, [a, _], g in assoc(a, :grader))
```
Comment (indocomsoft) [inline lib/cadet/assessments/assessments.ex]: Prefer `|> join(:left, [a, ...], g in assoc(a, :grader))`

File: lib/cadet/assessments/assessments.ex [hunk 4]
```diff
@@ -488,16 +488,17 @@ defmodule Cadet.Assessments do
...
 
     cond do
       role in @grading_roles ->
         students = Cadet.Accounts.Query.students_of(grader)
 
         answers =
           answer_query
-          |> join(:inner, [a], s in Submission, a.submission_id == s.id)
-          |> join(:inner, [a, _, _, s], t in subquery(students), t.id == s.student_id)
+          |> join(:inner, [_, _, _, s, _], t in subquery(students), t.id == s.student_id)
```
Comment (indocomsoft) [inline lib/cadet/assessments/assessments.ex]: Prefer `|> join(:inner, [..., s, _], t in subquery(students), t.id == s.student_id)`

File: lib/cadet/assessments/assessments.ex (hunk 2 above)
Comment (indocomsoft) [reply lib/cadet/assessments/assessments.ex]: Erm I think you missed this one

Comment (jiachen247) [reply lib/cadet/assessments/assessments.ex]: opps my bad! made the changes.

**Pull request merged**